src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -613,6 +630,52 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _currentIconType == _IconTypes.GetAt(3);
     }
 
+    hstring ProfileViewModel::BellStylePreview() const
+    {
+        const auto bellStyle = BellStyle();
+        const bool isAudibleSet = WI_IsFlagSet(bellStyle, BellStyle::Audible);
+        const bool isWindowSet = WI_IsFlagSet(bellStyle, BellStyle::Window);
+        const bool isTaskbarSet = WI_IsFlagSet(bellStyle, BellStyle::Taskbar);
+        if (bellStyle == Model::BellStyle::All || (isAudibleSet && isWindowSet && isTaskbarSet))
+        {
+            return RS_(L"Profile_BellStyleAll/Content");
+        }
+        else if (bellStyle == static_cast<Model::BellStyle>(0))
+        {
+            return RS_(L"Profile_BellStyleNone/Content");
+        }
+
+        std::vector<hstring> resultList;
+        resultList.reserve(3);
+        if (isAudibleSet)

```

Comment:
replace all these with


```suggestion
        if (WI_IsFlagSet(bellStyle, BellStyle::Audible))
```

and get rid of the temporary booleans. :)